calico/_includes/release-notes/_master-release-notes.mdx

```diff
@@ -7,7 +7,9 @@ releases are displayed.
 -->
 ### What's new {#whats-new-v0.1.0}
 
-#### New feature 1
+#### Support for Azure CNI with overlay networking
```

Comment:
We'll need to find the right home for this. I'm thinking that we'd be best replicating the structure from CE.